### PR TITLE
adblock: Added Netherlands as a region.

### PR DESCRIPTION
**Pull request by pcmulder**
Signed-off-by: Patrick Mulder <pcmulder89@gmail.com>

### CODE REVIEW

**Comment by zorun** [comment]
Hi, thanks for the PR.  You need to prefix the subject of the commit with the package (here `adblock:`), and also sign-off your commit.

**Comment by pcmulder** [comment]
This should do it I think. Thanks for the info.

**Comment by dibdot** [comment]
@pcmulder please increment PKG_RELEASE in the makefile as well.
Everything else looks good to me - thanks.

File: net/adblock/Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=adblock
-PKG_VERSION:=2.8.3
```
Comment (pcmulder) [inline net/adblock/Makefile]: On the master this showed 4, but here 3? Did I mess something up?

**Comment by hnyman** [comment]
Neither of your commits is valid, as they do not have proper commit titles and neither of them is signed off. The commits themselves should be signed as they get stored to git history, unlike this PR discussion.
You should combine the commits (squash them) and edit the commit message at the same time.

**Comment by hnyman** [comment]
And dibdot asked you to increase PKG_RELEASE not PKG_VERSION

**Comment by pcmulder** [comment]
Thank you for the feedback. I made some changes. Is this alright now?

**Pull request merged**